README.md

```diff
@@ -42,21 +42,11 @@ cs = jet_reconstruct(particles::AbstractVector{T}; algorithm = JetAlgorithm.Anti
   - `JetAlgorithm.Durham` the $e^+e-$ $k_\text{T}$ algorithm, also known as the Durham algorithm
   - `JetAlgorithm.EEKt` the $e^+e-$ generalised $k_\text{T}$ algorithm
 - `R` - the cone size parameter; no particles more geometrically distance than `R` will be merged (default 1.0; note this parameter is ignored for the Durham algorithm)
+- `p` - algorithm power which must be provided for generalised algorithms, for other algorithms this parameter is ignored
```

Comment:
Drop `p` and `strategy` descriptions.

We could link to https://juliahep.github.io/JetReconstruction.jl/stable/#Reconstruction-Interface as a "see here for additional options"